### ISSUE TITLE
Can't run on Windows 8.1 64bit, Node.js 0.12.4

### ISSUE THREAD

**Issue by stereokai** [caspervonb/amok#82]
@caspervonb This looks so exciting and I can't wait to start working with it, but I am getting this error on your react-simple example. What gives?

```
> @ start C:\Users\Tom\Dev\stereokai-player
> amok index.js --client chrome --compiler browserify --verbose --transform reactify --fast

net.js:633
    throw new TypeError('invalid data');
          ^
TypeError: invalid data
    at WriteStream.Socket.write (net.js:633:11)
    at C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\bin\amok.js:88:20
    at C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\lib\index.js:53:14
    at next (C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\node_modules\ware\lib\index.js:77:35
)
    at C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\node_modules\ware\node_modules\wrap-fn\ind
ex.js:121:18
    at Ware.<anonymous> (C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\node_modules\ware\node_m
odules\wrap-fn\index.js:47:16)
    at next (C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\node_modules\ware\lib\index.js:85:20
)
    at C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\node_modules\ware\node_modules\wrap-fn\ind
ex.js:121:18
    at Server.ready (C:\Users\Tom\AppData\Roaming\npm\node_modules\amok\lib\server.js:27:7)
    at Server.g (events.js:199:16)

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\
\bin\\npm-cli.js" "start"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! code ELIFECYCLE
npm ERR! @ start: `amok index.js --client chrome --compiler browserify --verbose --transform reactif
y --fast`
npm ERR! Exit status 1
```

**Comment by stereokai** [comment]
A quick look shows that `ware.run()` returns an error for one of the plugins, implying that maybe something is wrong with the configuration?

**Comment by caspervonb** [comment]
The examples have not been updated to 1.x yet.

**Comment by stereokai** [comment]
*You are the man*

Thanks.

**Comment by stereokai** [comment]
@caspervonb Actually, Amok now opens up a new tab and serves the example, but if I edit index.js, nothing updates in the browser.

I appreciate your attentiveness.

**Comment by caspervonb** [comment]
Your browser has to either be already be listening on the port specified by the `--debug-port` option (default 9222), or amok has to spawn a new process. But if the browser is already open only a new window will be spawned, so either provide a custom profile via the `CHROME_FLAGS` environment variable ala

```
set CHROME_FLAGS="user-data-dir=C:\amok_chrome_profile"
```

Or make sure chrome is not running before starting chrome, It's in the manual ;)

**Comment by stereokai** [comment]
Man, I apologize. I should have paid more attention in class.

When I did that and ran `npm start`, Chrome has created a new user profile. What have I done wrong again?

**Comment by caspervonb** [comment]
By either providing a user profile or ensuring a fresh start, you should be good to go.

**Comment by stereokai** [comment]
I *have* provided the path to my own user profile, as so:
```
--user-datadir=C:\Users\Tom\AppData\Local\Google\Chrome\User Data\Default
```
Isn't that the correct path?

By the way, I also received this message in my terminal:
```
[6936:7096:0610/135726:ERROR:bluetooth_adapter_win.cc(102)] NOT IMPLEMENTED
Download the React DevTools for a better development experience: https://fb.me/react-devtools
```

**Comment by caspervonb** [comment]
If chrome is your day to day browser, you'd want a separate user profile for development to make sure it'll be in debugger mode

**Comment by stereokai** [comment]
Pardon me, I couldn't find anything about Chrome Debugger Mode. Could you kindly tell me about it?

**Comment by stereokai** [comment]
I mean, are you talking about the devtools:

![image](https://cloud.githubusercontent.com/assets/2685357/8081290/48d9aed0-0f7c-11e5-9d7f-535b1e6b4b24.png)

That's what I could bring up trying to find chrome debugger mode. Please expand why is it important to have a separate profile? Thank you.

**Comment by caspervonb** [comment]
Was referring to remote debugging server, enabled with `--remote-debugging-enabled=PORT` on chrome like browsers.

TLDR;
If you use Chrome as your every day browser, I'd recommended that you use a separate profile for development with amok to ensure that you get separate processes for development and normal browser usage.

```sh
set CHROME_FLAGS="--user-data-dir=path/to/anything"
```

Chrome will create a profile in that directory and use it.

**Comment by stereokai** [comment]
Got it. Thanks buddy.

**Comment by caspervonb** [comment]
This was resolved way back in the archive branch https://github.com/caspervonb/amok/commit/f629bd76ff491daa81a99580cc38e11b3b00f8ba

The default behavior since 1.0 is that each browser is spawned with its own unique profile.
